django/contrib/gis/gdal/srs.py

```diff
@@ -42,7 +46,7 @@ class SpatialReference(GDALBase):
     """
     destructor = capi.release_srs
 
-    def __init__(self, srs_input='', srs_type='user'):
+    def __init__(self, srs_input='', srs_type='user', axis_order='traditional'):
```

Comment:
I wasn't sure if we should set 'traditional' axis order by default or leave GDAL 3 default axis order. But since leaving GDAL default axis order is breaking and I didn't find any pros in it, I'm fine with using 'traditional' axis order as default. If we want we can change it later.

Reply:
I think `'traditional'` by default makes sense - everything should keep working then. Nothing worse than an underlying library upgrade unexpectedly screwing everything up. Let use of this new behaviour be an advanced option to those that are aware of the consequences.

Also, wouldn't we need to document this axis stuff everywhere otherwise?

Should we also only accept something like `'authority'` as an alternate value rather than leaving it open?